modules/styleguide/assets/js/frontend/components/global/element-title.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+const ElementTitle = styled.p`
+	color: #515962;
```

Comment:
Always use variables for colors, here and other files.

Reply:
Fixed